spirv_cross.cpp

```diff
@@ -998,6 +998,12 @@ uint64_t Compiler::get_decoration_mask(uint32_t id) const
 	return dec.decoration_flags;
 }
 
+bool Compiler::is_decoration_set(uint32_t id, spv::Decoration decoration) const
```

Comment:
get_decoration_mask does something very similar, do we need another interface to check decorations?